### ISSUE TITLE
AN/DN clickables not working correctly 

### ISSUE THREAD

**Issue by Torih** [linuxgurugamer/ksp-precisenode#7]
Both the AN/DN buttons are not placing the nodes in the correct place for the target you have selected. I think it places them on the AN/DN for the body you are orbiting.

https://www.dropbox.com/s/wxbbux5trv7zcof/AN-DN.jpg?dl=0 Just shows where the node is placed when clicking AN when targetting Minmus

**Comment by linuxgurugamer** [comment]
Was it this way in 1.3.1?

**Comment by Torih** [comment]
Yeah, it was in 1.3.1, forget when it started its been like this a while.

**Comment by linuxgurugamer** [comment]
Ok, then I'm going to leave it for now, since it's not a new bug.  I'll get back to it when all updates are done
Thanks
